src/conduits/RwaOutputConduit3.sol

```diff
@@ -354,7 +354,7 @@ contract RwaOutputConduit3 {
      * @notice Flushes out the specified amount of DAI to `quitTo` address.
      * @param wad The DAI amount.
      */
-    function _doQuit(uint256 wad) internal {
+    function _doQuit(uint256 wad) internal { // TB - as discussed I think the pattern of having internal functions right after their wrappers is nicer - https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L232
```

Comment:
Since it's just a matter of preference, this is one of the coding standard that we established in CES.